README.md

```diff
@@ -1,3 +1,77 @@
-# dra-fair-teaching
+# Reproducible & FAIR Teaching Materials
 
-[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
\ No newline at end of file
+[![Quarto Publish](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml/badge.svg)](https://github.com/lnnrtwttkhn/dra-fair-teaching/actions/workflows/publish.yml)
+
+## About
+
+Slides for talk on "Reproducible & FAIR Teaching Materials" by Esther Plomp and Lennart Wittkuhn.
+Prepared for the Digital Research Academy train-the-trainer program.
+
+## Requirements
+
+- [Quarto](https://quarto.org/)
+- [renv](https://rstudio.github.io/renv/articles/renv.html)
```

Comment:
Should we add R / RStudio?

Reply:
good idea! added in https://github.com/lnnrtwttkhn/dra-fair-teaching/pull/13/commits/255182b475ef720a50eeeafa8cf3fd2d0b721ab7